app/src/main/java/com/babylon/wallet/android/presentation/account/settings/AccountSettingsScreen.kt

```diff
@@ -143,7 +119,10 @@ fun AccountSettingsScreen(
         faucetState = state.faucetState,
         isXrdLoading = state.isFreeXRDLoading,
         onHideAccount = {
-            showHideAccountPrompt = true
+            scope.launch {
```

Comment:
maybe we can define 
`val hideSheetAction: () -> Unit = {
        scope.launch {
            bottomSheetState.hide()
            viewModel.resetBottomSheetContent()
        }
    }` and use it throughout this composable, as this action is repeated multiple times

Reply:
Sounds good, thanks!